documentation/get-started/tutorials/advanced-setup-qfc.en.md

```diff
@@ -61,11 +61,7 @@ It requires your database to be publicly accessible, and credentials must be sav
     When using `direct database access`, QFieldCloud will directly edit data on the PostGIS database. This will only work with a reliable internet connection in the field, but has the advantage that all data is directly visible to all users and allows to use any PostGIS specific setup (triggers, generated fields, etc).
 
 !!! note
-    When using `offline editing`, QField will work on a local copy of the database in a GeoPackage, which will be synced by QFieldCloud to the original database.
-    This is the best choice if the connection in the field is not reliable.
-    Changes will only be visible to users once they sync to QFieldCloud.
-    As a local copy is created, advanced PostGIS features will not be available on QField.
-    Just like for regular GeoPackages, if you define relationships, it is recommended to use UUIDs instead of integer primary keys to avoid conflicts if multiple users create data at the same time.
+    When using `offline editing`, QField will work on a local copy of the database in a GeoPackage, which will be synced by QFieldCloud to the original database once synchronized by the user. This is the best choice if the connection in the field is not reliable. Changes will only be visible to users once the synchronization via QFieldCloud has been applied on the different devices. As a local copy is created, advanced PostGIS operations (like triggers) will not be available on QField. Just like for regular GeoPackages, if relationships are defined, it is recommended to use a UUID field instead of the fid as the primary key to avoid conflicts if multiple users create data at the same time.
```

Comment:
```suggestion
    When using `offline editing`, QField will work on a local copy of the database in a GeoPackage, which will be synced by QFieldCloud to the original database once synchronized by the user.
    This is the best choice if the connection in the field is not reliable
     Changes will only be visible to users once the synchronization via QFieldCloud has been applied on the different devices.
    As a local copy is created, advanced PostGIS operations (like triggers) will not be available on QField.
    Just like for regular GeoPackages, if relationships are defined, it is recommended to use a UUID field instead of the fid as the primary key to avoid conflicts if multiple users create data at the same time.
```